Dockerfile

```diff
@@ -58,15 +61,12 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
 ## Remove any existing configurations
 RUN rm -f /etc/httpd/conf.d/*
 
-COPY docker-assets/entrypoint /usr/bin
-
 EXPOSE 80
 
 WORKDIR /etc/httpd
 
-RUN systemctl enable dbus httpd
+RUN systemctl enable httpd
 
-VOLUME [ "/sys/fs/cgroup" ]
```

Comment:
We still need this line to get things running in minishift, will adding it back cause problems in OCP?